All EIPIP Meetings/Meeting 012.md

```diff
@@ -0,0 +1,364 @@
+
+(Matt) - Yeah, I know you guys have thought a lot about this. So I'll think about it on my own, but that generally makes a lot of sense.
+
+(James) - Yeah, and I appreciate the questions and critical questions on that. Because we can't adequately define how we've made decisions when we need to be remaking and rethinking them.
+
+(Pooja) - I'm sorry about the screen-sharing issue. Nex ttime I'll get it fixed before we start the metting.
+
+(Edson) - ONe final comment about that, I think that currently the hardfork meta EIP is its own EIP seperate from EIP 1 and EFI tracker is its own EIP seperate from EIP 1, so it looks like all all hard-fork coordination EIPs are their own EIP. SO that's another reason for why I'm inclined to giving some EIP the hardfork coordination process.
+
+(Alita) - Are you saying that we should merge all those different standards?
+
+(Edson) - So the new hardfork coordination process I think that we should create its own EIP and so when we update it we can supercede it. But for the new statuses we should make a PR for the EIP 1.
+
+(Pooja) - Yeah, I think it would make sense, I remeber this conversation in one fo the previous meetings that we requested that EIp 1 should not be updated so freqeuntly. Historically the hardfork process has changed, so it makes sense that we provide a link to the EFI process is explained and add it to EIP 1 instead of adding it directly to EIP 1. Any more questions or comments on this topic? We are almost at the end of the meeting.
+
+(James) - I think it would be good to have the EIP written for the network coordination, the network upgrade process before we do the EIP 1 for statuses. That actually make more sense. So the next step is to actually make an EIP for documenting the network upgrade status as it stands. That's better so I'm glad we didn't do the other way that I think about it.
+
+(Pooja) - So we can add it as an action item for now, and we can see in the next meeting if we are done with that, then people can go ahead and create the PR for updating EIP 1
+
+## 4.1. Decisions
+
+- *12.4* - Link to the live EFI document through EIP 1 rather than continuously updating EIP 1; EIP 1 can be used as a hub for statuses instead.
+
+# 5. Discuss the EIP-1 Brainstorming Document. 

```

Comment:
replace 5. and add agenda number

Reply:
Looks like this is a separate topic "emergency communications" and not under the head of EIP-1 brainstorming.